package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.1.3",
   "homepage": "https://github.com/aframevr/aframe-core",
   "license": "MIT",
-  "main": "src/index.js",
+  "main": "dist/aframe-core.js",
```

Comment:
Maintainers, as in us? Dev against which libraries exactly?

It should probably be done, but it's low priority for me as it only targets a handful of people (us), where some of us already know how to link.